### ISSUE TITLE
Function fast_scraper_schedules() Does Not Appear to be Working

### ISSUE THREAD

**Issue by JoeMarino2021** [nflverse/nflfastR#372]
When I execute `nflfastR::fast_scraper_schedules(2022)`, I get this output.

```
Warning: Either the requested season 2022 is invalid or no data available at this time!
# A tibble: 0 × 0
```

I check Lee Sharpe's data, and it appears to be there.

`nflreadr::load_schedules(2022)`

```
Note: nflreadr caches (i.e., stores a saved version) data by default.
If you expect different output try one of the following:
ℹ Restart your R Session or
ℹ Run `nflreadr::.clear_cache()`.
To disable this warning, run `options(nflreadr.verbose = FALSE)` or add it to your .Rprofile
This message is displayed once every 8 hours.
── nflverse games and schedules ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
# A tibble: 272 × 45
   game_id season game_…¹  week gameday weekday gamet…² away_…³ away_…⁴ home_…⁵ home_…⁶ locat…⁷ result total overt…⁸ old_g…⁹  gsis nfl_d…˟ pfr     pff espn  away_…˟ home_…˟ away_…˟ home_…˟ sprea…˟ away_…˟ home_…˟ total…˟ under…˟ over_…˟ div_g…˟ roof 
   <chr>    <int> <chr>   <int> <chr>   <chr>   <chr>   <chr>     <int> <chr>     <int> <chr>    <int> <int>   <int> <chr>   <int> <chr>   <chr> <int> <chr>   <int>   <int>   <int>   <int>   <dbl>   <int>   <int>   <dbl>   <int>   <int>   <int> <chr>
 1 2022_0…   2022 REG         1 2022-0… Thursd… 20:20   BUF          31 LA           10 Home       -21    41       0 202209…    NA NA      2022…    NA 4014…       7       7    -117     106    -1      -112     101    51.5    -101    -111       0 dome 
 2 2022_0…   2022 REG         1 2022-0… Sunday  13:00   NO           27 ATL          26 Home        -1    53       0 202209…    NA NA      2022…    NA 4014…       7       7    -228     201    -6      -103    -107    43.5    -104    -108       1 NA   
 3 2022_0…   2022 REG         1 2022-0… Sunday  13:00   CLE          26 CAR          24 Home        -2    50       0 202209…    NA NA      2022…    NA 4014…       7       7     114    -125     2.5    -112     102    42.5    -113     100       0 outd…
 4 2022_0…   2022 REG         1 2022-0… Sunday  13:00   SF           10 CHI          19 Home         9    29       0 202209…    NA NA      2022…    NA 4014…       7       7    -248     217    -6.5     100    -111    38      -103    -109       0 outd…
 5 2022_0…   2022 REG         1 2022-0… Sunday  13:00   PIT          23 CIN          20 Home        -3    43       1 202209…    NA NA      2022…    NA 4014…       7       7     264    -308     7.5    -119     107    44.5     101    -113       1 outd…
 6 2022_0…   2022 REG         1 2022-0… Sunday  13:00   PHI          38 DET          35 Home        -3    73       0 202209…    NA NA      2022…    NA 4014…       7       7    -240     211    -6      -110    -100    49      -117     104       0 dome 
 7 2022_0…   2022 REG         1 2022-0… Sunday  13:00   IND          20 HOU          20 Home         0    40       1 202209…    NA NA      2022…    NA 4014…       7       7    -290     250    -7      -103    -107    46      -110    -102       1 NA   
 8 2022_0…   2022 REG         1 2022-0… Sunday  13:00   NE            7 MIA          20 Home        13    27       0 202209…    NA NA      2022…    NA 4014…       7       7     140    -155     3      -104    -106    47      -109    -103       1 outd…
 9 2022_0…   2022 REG         1 2022-0… Sunday  13:00   BAL          24 NYJ           9 Home       -15    33       0 202209…    NA NA      2022…    NA 4014…       7       7    -265     231    -6.5    -103    -107    44      -109    -103       0 outd…
10 2022_0…   2022 REG         1 2022-0… Sunday  13:00   JAX          22 WAS          28 Home         6    50       0 202209…    NA NA      2022…    NA 4014…       7       7     143    -159     3      -101    -110    43.5    -110    -102       0 outd…
# … with 262 more rows, 12 more variables: surface <chr>, temp <int>, wind <int>, away_qb_id <chr>, home_qb_id <chr>, away_qb_name <chr>, home_qb_name <chr>, away_coach <chr>, home_coach <chr>, referee <chr>, stadium_id <chr>, stadium <chr>, and
#   abbreviated variable names ¹​game_type, ²​gametime, ³​away_team, ⁴​away_score, ⁵​home_team, ⁶​home_score, ⁷​location, ⁸​overtime, ⁹​old_game_id, ˟​nfl_detail_id, ˟​away_rest, ˟​home_rest, ˟​away_moneyline, ˟​home_moneyline, ˟​spread_line, ˟​away_spread_odds,
#   ˟​home_spread_odds, ˟​total_line, ˟​under_odds, ˟​over_odds, ˟​div_game
# ℹ Use `print(n = ...)` to see more rows, and `colnames()` to see all variable names
```

I check the function on 2021 to see if it might be deprecated.

 `nflfastR::fast_scraper_schedules(2021)`

```
# A tibble: 285 × 17
   game_id         season game_type  week gameday    weekday  gametime away_team home_team away_score home_score home_result stadium                 location roof     surface   old_game_id
   <chr>            <int> <chr>     <int> <chr>      <chr>    <chr>    <chr>     <chr>          <int>      <int>       <int> <chr>                   <chr>    <chr>    <chr>     <chr>      
 1 2021_01_DAL_TB    2021 REG           1 2021-09-09 Thursday 20:20    DAL       TB                29         31           2 Raymond James Stadium   Home     outdoors grass     2021090900 
 2 2021_01_PHI_ATL   2021 REG           1 2021-09-12 Sunday   13:00    PHI       ATL               32          6         -26 Mercedes-Benz Stadium   Home     NA       fieldturf 2021091200 
 3 2021_01_PIT_BUF   2021 REG           1 2021-09-12 Sunday   13:00    PIT       BUF               23         16          -7 New Era Field           Home     outdoors astroturf 2021091201 
 4 2021_01_NYJ_CAR   2021 REG           1 2021-09-12 Sunday   13:00    NYJ       CAR               14         19           5 Bank of America Stadium Home     outdoors grass     2021091202 
 5 2021_01_MIN_CIN   2021 REG           1 2021-09-12 Sunday   13:00    MIN       CIN               24         27           3 Paul Brown Stadium      Home     outdoors grass     2021091203 
 6 2021_01_SF_DET    2021 REG           1 2021-09-12 Sunday   13:00    SF        DET               41         33          -8 Ford Field              Home     dome     fieldturf 2021091204 
 7 2021_01_JAX_HOU   2021 REG           1 2021-09-12 Sunday   13:00    JAX       HOU               21         37          16 NRG Stadium             Home     NA       grass     2021091205 
 8 2021_01_SEA_IND   2021 REG           1 2021-09-12 Sunday   13:00    SEA       IND               28         16         -12 Lucas Oil Stadium       Home     NA       fieldturf 2021091206 
 9 2021_01_ARI_TEN   2021 REG           1 2021-09-12 Sunday   13:00    ARI       TEN               38         13         -25 Nissan Stadium          Home     outdoors grass     2021091207 
10 2021_01_LAC_WAS   2021 REG           1 2021-09-12 Sunday   13:00    LAC       WAS               20         16          -4 FedExField              Home     outdoors grass     2021091208 
# … with 275 more rows
# ℹ Use `print(n = ...)` to see more rows
```

So, the function loaded 2021 OK.  Lee Sharpe shows data for 2022.

Am I doing something wrong?  This code worked for me last year; I don't see anything in the documentation that would indicate I need to change something on my end.  Is the function `nflfastR::fast_scraper_schedules()` no longer functioning for 2022?

Thanks.

**Comment by mrcaseb** [comment]
Can you please provide the output of `nflreadr::nflverse_sitrep()`?

**Comment by JoeMarino2021** [comment]
I noticed that some packages from NFLVerse were not installed.  I installed them.  Then I tried the command again.  It still didn't work.  Then I loaded the library NFLVerse.  I tried the command again.  Still no joy.  See below for all of the efforts.

Thanks for the reply.  Let me know if you need anything else.


`> nflreadr::nflverse_sitrep()`
```
ℹ You've asked for the packages nflseedR, nfl4th, nflplotR, and nflverse which are not installed. 
They are skipped.
── System Info ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
• R version 4.2.1 (2022-06-23 ucrt)   • Running under: Windows 10 x64 (build 19043)
── nflverse Packages ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
• nflreadr (1.2.0)
• nflfastR (4.3.0)
── nflverse Dependencies ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
• cachem      (1.0.6)   • globals    (0.15.1)   • pkgconfig  (2.0.3)    
• cli         (3.3.0)   • glue       (1.6.2)    • progressr  (0.10.1)   
• codetools   (0.2-18)  • janitor    (2.1.0)    • purrr      (0.3.4)    
• cpp11       (0.4.2)   • jsonlite   (1.8.0)    • R6         (2.5.1)    
• curl        (4.3.2)   • lattice    (0.20-45)  • rappdirs   (0.3.3)    
• data.table  (1.14.2)  • lifecycle  (1.0.1)    • rlang      (1.0.4)    
• digest      (0.6.29)  • listenv    (0.8.0)    • snakecase  (0.11.0)   
• dplyr       (1.0.9)   • lubridate  (1.8.0)    • stringi    (1.7.8)    
• ellipsis    (0.3.2)   • magrittr   (2.0.3)    • stringr    (1.4.0)    
• fansi       (1.0.3)   • Matrix     (1.4-1)    • tibble     (3.1.8)    
• fastmap     (1.1.0)   • memoise    (2.0.1)    • tidyr      (1.2.0)    
• fastrmodels (1.0.2)   • mgcv       (1.8-40)   • tidyselect (1.1.2)    
• furrr       (0.3.0)   • nlme       (3.1-157)  • utf8       (1.2.2)    
• future      (1.27.0)  • parallelly (1.32.1)   • vctrs      (0.4.1)    
• generics    (0.1.3)   • pillar     (1.8.0)    • xgboost    (1.6.0.1)  
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
```
`> install.packages("nflverse")`
```
WARNING: Rtools is required to build R packages but is not currently installed. Please download and install the appropriate version of Rtools before proceeding:

https://cran.rstudio.com/bin/windows/Rtools/
Installing package into ‘C:/Users/JOE/AppData/Local/R/win-library/4.2’
(as ‘lib’ is unspecified)
also installing the dependencies ‘proto’, ‘magick’, ‘gsubfn’, ‘nfl4th’, ‘nflplotR’, ‘nflseedR’

trying URL 'https://cran.rstudio.com/bin/windows/contrib/4.2/proto_1.0.0.zip'
Content type 'application/zip' length 472368 bytes (461 KB)
downloaded 461 KB

trying URL 'https://cran.rstudio.com/bin/windows/contrib/4.2/magick_2.7.3.zip'
Content type 'application/zip' length 19428420 bytes (18.5 MB)
downloaded 18.5 MB

trying URL 'https://cran.rstudio.com/bin/windows/contrib/4.2/gsubfn_0.7.zip'
Content type 'application/zip' length 358135 bytes (349 KB)
downloaded 349 KB

trying URL 'https://cran.rstudio.com/bin/windows/contrib/4.2/nfl4th_1.0.2.zip'
Content type 'application/zip' length 360501 bytes (352 KB)
downloaded 352 KB

trying URL 'https://cran.rstudio.com/bin/windows/contrib/4.2/nflplotR_1.1.0.zip'
Content type 'application/zip' length 2157149 bytes (2.1 MB)
downloaded 2.1 MB

trying URL 'https://cran.rstudio.com/bin/windows/contrib/4.2/nflseedR_1.1.0.zip'
Content type 'application/zip' length 472496 bytes (461 KB)
downloaded 461 KB

trying URL 'https://cran.rstudio.com/bin/windows/contrib/4.2/nflverse_1.0.2.zip'
Content type 'application/zip' length 92545 bytes (90 KB)
downloaded 90 KB

package ‘proto’ successfully unpacked and MD5 sums checked
package ‘magick’ successfully unpacked and MD5 sums checked
package ‘gsubfn’ successfully unpacked and MD5 sums checked
package ‘nfl4th’ successfully unpacked and MD5 sums checked
package ‘nflplotR’ successfully unpacked and MD5 sums checked
package ‘nflseedR’ successfully unpacked and MD5 sums checked
package ‘nflverse’ successfully unpacked and MD5 sums checked

The downloaded binary packages are in
	C:\Users\JOE\AppData\Local\Temp\RtmpKAurKI\downloaded_packages
```
`> nflreadr::nflverse_sitrep()`
```
── System Info ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
• R version 4.2.1 (2022-06-23 ucrt)   • Running under: Windows 10 x64 (build 19043)
── nflverse Packages ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
• nflreadr (1.2.0)  • nflseedR (1.1.0)  • nflplotR (1.1.0)  
• nflfastR (4.3.0)  • nfl4th   (1.0.2)  • nflverse (1.0.2)  
── nflverse Dependencies ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
• askpass     (1.1)     • gtable      (0.3.0)    • progressr    (0.10.1)   
• bit         (4.0.4)   • hms         (1.1.1)    • proto        (1.0.0)    
• bit64       (4.0.5)   • httr        (1.4.3)    • purrr        (0.3.4)    
• cachem      (1.0.6)   • isoband     (0.2.5)    • R6           (2.5.1)    
• cli         (3.3.0)   • janitor     (2.1.0)    • rappdirs     (0.3.3)    
• clipr       (0.8.0)   • jsonlite    (1.8.0)    • RColorBrewer (1.1-3)    
• codetools   (0.2-18)  • labeling    (0.4.2)    • Rcpp         (1.0.9)    
• colorspace  (2.0-3)   • lattice     (0.20-45)  • readr        (2.1.2)    
• cpp11       (0.4.2)   • lifecycle   (1.0.1)    • rlang        (1.0.4)    
• crayon      (1.5.1)   • listenv     (0.8.0)    • rstudioapi   (0.13)     
• curl        (4.3.2)   • lubridate   (1.8.0)    • scales       (1.2.0)    
• data.table  (1.14.2)  • magick      (2.7.3)    • snakecase    (0.11.0)   
• digest      (0.6.29)  • magrittr    (2.0.3)    • stringi      (1.7.8)    
• dplyr       (1.0.9)   • MASS        (7.3-57)   • stringr      (1.4.0)    
• ellipsis    (0.3.2)   • Matrix      (1.4-1)    • sys          (3.4)      
• fansi       (1.0.3)   • memoise     (2.0.1)    • tibble       (3.1.8)    
• farver      (2.1.1)   • mgcv        (1.8-40)   • tidyr        (1.2.0)    
• fastmap     (1.1.0)   • mime        (0.12)     • tidyselect   (1.1.2)    
• fastrmodels (1.0.2)   • munsell     (0.5.0)    • tzdb         (0.3.0)    
• furrr       (0.3.0)   • nlme        (3.1-157)  • utf8         (1.2.2)    
• future      (1.27.0)  • openssl     (2.0.2)    • vctrs        (0.4.1)    
• generics    (0.1.3)   • parallelly  (1.32.1)   • viridisLite  (0.4.0)    
• ggplot2     (3.3.6)   • pillar      (1.8.0)    • vroom        (1.5.7)    
• globals     (0.15.1)  • pkgconfig   (2.0.3)    • withr        (2.5.0)    
• glue        (1.6.2)   • prettyunits (1.1.1)    • xgboost      (1.6.0.1)  
• gsubfn      (0.7)     • progress    (1.2.2)      
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
```
`> nflfastR::fast_scraper_schedules(2022)`
```
Warning: Either the requested season 2022 is invalid or no data available at this time!
# A tibble: 0 × 0
```
`> library(nflverse)`
```
── Attaching packages ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── nflverse 1.0.2 ──
✔ nflfastR 4.3.0     ✔ nflreadr 1.2.0
✔ nflseedR 1.1.0     ✔ nflplotR 1.1.0
✔ nfl4th   1.0.2     
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── Ready to go! ──
```
`> nflfastR::fast_scraper_schedules(2022)`
```
Warning: Either the requested season 2022 is invalid or no data available at this time!
# A tibble: 0 × 0
```

**Comment by mrcaseb** [comment]
The released version of nflfastR is 4.4.0 for which the command would work. Please restart your session and try again. If you are still running 4.3.0 for some reason please run 
`nflverse::nflverse_update()`. It will provide you install instructions for outdated versions.  

Did you follow the issue instructions or just deleted them before you created that issue?

**Comment by JoeMarino2021** [comment]
I should have thought to try to update the RStudio packages.  I not long ago updated NFLFastR and assumed I was ready to go for the season.  Turns out you guys put out a dot release that I did not have.

Sorry about this.  I could have spared you some effort if I would have checked for updates.  I assumed everything was current.

Now all is working again.

Thanks for the most wonderful R package.  The season is even more fun because of it.
